database/migrations/2018_03_07_143627_create_groups_members_table.php

```diff
@@ -20,7 +20,7 @@ public function up()
             $table->uuid('user_id');
             $table->uuid('role_id')->nullable();
             $table->uuid('semester_id');
-            // On permet ici que le semestre soit égal à 0 pour ne convenir à aucun semestre
+            // We allow here the semester to be equal to 0 (Between two semesters for example).
```

Comment:
Wrong translation here:
`We allow here the semester to be nullable so the membership could be for ever`

Reply:
To be null ;) (=nullable)